pyracf/common/irrsmo00.c

```diff
@@ -13,6 +13,19 @@ typedef struct {
         char str[8];
 } VarStr_T;
 
+void null_byte_fix(char* str, unsigned int str_len) {
+   for (int i = 1; i < str_len; i++){
```

Comment:
Are you able to add a comment explaining what this is for?

Reply:
I don't see why not, I will try to do so to explain this better.

Reply:
Also, I suggest changing the word `whitespace` to `space characters` or something like that instead if this is the case.

Reply:
I suppose we don't know what the exact EBCDIC characters are returned by the RACF command as I've only ever copy/pasted what's rendered by ISPF. I will change the wording to make it generic enough to relay that, at least until we can discuss with RACF development to see if it is a "bug" or not.

Reply:
```c
// This function changes any null character not preceded by '>' to a blank character.
// This is a workaround for an issue where the xml returned by IRROSMO00 sometimes
// includes null characters instead of properly encoded text, which causes the returned
// xml to be truncated.
```